docs/en/connector-v2/sink/Kafka.md

```diff
@@ -84,6 +84,10 @@ If name is set as the key, then the hash value of the name column will determine
 
 If not set partition key fields, the null message key will be sent to.
 
+The format of the message key is json, If name is set as the key, for example '{"name":"Jack"}'.
```

Comment:
Hi, I don't get the mean about this part.

Reply:
This means that after setting the "partition_key_fields" attribute, the format of the kafka message key is json, and an example is listed.

Reply:
Maybe you should change to `If the format of the message is json, and the name field be setted as the partition_key_fields, for example '{"name":"Jack"}'. Then `Jack` will be used for partition`.

Reply:
Maybe you should change to `If the format of the message is json, and the name field be setted as the partition_key_fields, for example '{"name":"Jack"}'. Then `Jack` will be used for partition`.

Reply:
This is not the format of the message. This is the key format of the message written to kafka after setting the "partition_key_fields" attribute. For example, if the "partition_key_fields" attribute is set to name, the key of the message kafka message should be '{"name": "Jack"}'.

Reply:
I checked the code, you are right.